docker/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM node:alpine
+
+RUN mkdir /app
+COPY server/index.js /app
+COPY server/package.json /app
+
+WORKDIR /app
+RUN yarn
+
+ENV GRAPHQLPLAYGROUND_PORT 8080
+ENV GRAPHQLPLAYGROUND_HOST "http://localhost:3000/graphql"
```

Comment:
I think this should be `GRAPHQLPLAYGROUND_API_ENDPOINT` or `GRAPHQLPLAYGROUND_ENDPOINT`